pkg/resources/kafka/configmap_test.go

```diff
@@ -615,7 +615,6 @@ zookeeper.connect=example.zk:2181/`,
 
 		t.Run(test.testName, func(t *testing.T) {
 			mockClient := mocks.NewMockClient(mockCtrl)
-			mockClient.EXPECT().Get(gomock.Any(), gomock.Any(), gomock.Any()).Return(nil)
```

Comment:
This might not be needed and only a remainder after cherry picking from our Adobe fork. After the API module includes required changes, I will recheck and fix any issues while validating.